### PR TITLE
Improve: 避免重复添加`/`

### PR DESCRIPTION
**Pull request by oiuv**
目前代码存在重复添加`/`的问题，如果运行时配置文件指定路径加了`/`，结果`author_file`参数传的`__MASTER_FILE__`会多一个`/`。

### CODE REVIEW

File: src/vm/internal/base/interpret.cc [hunk 1]
```diff
@@ -489,7 +489,7 @@ void free_some_svalues(svalue_t *v, int num) {
 char *add_slash(const char *const str) {
   char *tmp;
 
-  if (str[0] == '<' && strcmp(str + 1, "function>") == 0) {
+  if (str[0] == '/' || str[0] == '<' && strcmp(str + 1, "function>") == 0) {
```
Comment (thefallentree) [inline src/vm/internal/base/interpret.cc:492]: what is the logic here?  || && is a bit confusing.

I think you should just add another check in front.

Comment (oiuv) [reply src/vm/internal/base/interpret.cc:492]: &&优先级高于||嘛，等等我加个括号()吧。

Comment (oiuv) [reply src/vm/internal/base/interpret.cc]: 增加了括号

Comment (oiuv) [reply src/vm/internal/base/interpret.cc]: > what is the logic here? || && is a bit confusing.
> 
> I think you should just add another check in front.

如果不过滤`/`，对`author_file`，获取的MASTER_OB文件路径是`//`开头，除非在config配置中不带`/`：
```c
Loading simul_efun file : /system/kernel/simul_efun
Loading master file: /system/kernel/master
author_file : //system/kernel/master
domain_file : /
Accepting telnet connections on 0.0.0.0:4002.
Initializations complete.
```

**Pull request closed without merging**